Doc/library/getopt.rst

```diff
@@ -7,11 +7,6 @@
 
 **Source code:** :source:`Lib/getopt.py`
 
-.. deprecated:: 3.13
-   The :mod:`getopt` module is :term:`soft deprecated` and will not be
-   developed further; development will continue with the :mod:`argparse`
-   module.
-
 .. note::
```

Comment:
I would convert the note in normal paragraph. Also, it should refer to `optparse` as an object-oriented and extensible alternative, not only to `argparse` (which is not an equivalent).

Reply:
I ended up moving this page back to the "Superseded" section of the docs, as it really is a niche use case when anyone should be reaching for this. The use of the note syntax here reflects that "You almost certainly don't want this module" status.

I agree about referring readers to optparse in the note, though.